test/libsolidity/syntaxTests/types/var_type_suggest.sol

```diff
@@ -10,7 +10,6 @@ contract C {
         var i = 31415;
         var t = "string";
         var g2 = g;
-        var myblockhash = block.blockhash;
```

Comment:
This was removed, because the type is not `nameable()` and calls `containsNestedMapping()`

The old behavior can likely be fixed by making the first error as `fatal`. But since it deals with a deprecated feature, I removed it.